control.lua

```diff
@@ -108,6 +114,14 @@ script.on_event(ev.on_entity_cloned, ctron.on_entity_cloned)
 script.on_event({ev.on_entity_destroyed, ev.script_raised_destroy}, ctron.on_entity_destroyed)
 
 script.on_event(ev.on_runtime_mod_setting_changed, ctron.mod_settings_changed)
+
+script.on_event(ev.on_player_used_spider_remote, (function(event)
+    if global.spider_remote_toggle and event.vehicle.name == "constructron" then
```

Comment:
is the vehicle name the actual prototype name or just the name of the vehicle that can be edited by users too?

Reply:
It's the prototype name but I just tested that as I don't think I've ever renamed something and it worked still.